### DISCUSSION TITLE
Welcome to bhyve-home-assistant Discussions!

### DISCUSSION THREAD

**Discussion by sebr** [sebr/bhyve-home-assistant#50] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by brent20** [comment]
Hello! Thank you for working on this custom component! I'll look at finally getting around to installing HACS and enabling it.

Why not submit this as an official integration with Home Assistant though?

Thanks!

**Reply by sebr** [reply]
Thank you @brent20

The main reason is because I have not had much time to support the requirements to be bundled (eg UI configuration support)
